### ISSUE TITLE
multifeed#removeFeed()

### ISSUE THREAD

**Issue by telamon** [noffle/multifeed#19]
**I want to remove a feed in order to:**
1. stop sharing selected content.
2. free up disk-space
3. prevent selected content from being downloaded again (blacklist a feed)

**I also want to pause replication for a feed in order to:**

4. keep downloaded content and temporarily save bandwidth.
5. prioritize low-availablity feeds

`3`, `4`, `5` can be implemented using [application defined API once available](https://github.com/noffle/multifeed/issues/2)

Perliminary proposal without changing current feed-storage scheme:

- [ ] call `feed.clear()` on 'removed' feed to reclaim disk-space
- [ ] Add 'removed' feeds public key to a blacklist array.
- [ ] Use `multifeed#_storage` to persist blacklist to `NOSYNC` file
- [ ] Ensure that `NOSYNC` file is loaded properly on start if exists
- [ ] For multifeed v1/master patch [L136](https://github.com/noffle/multifeed/blob/master/index.js#L136) & [L204](https://github.com/noffle/multifeed/blob/master/index.js#L204) to filter blacklisted feeds 
- [ ] For multifeed v2.x patch [L192](https://github.com/telamon/multifeed/blob/2.x/index.js#L192) to filter blacklisted feeds.

**Comment by substack** [comment]
This would be a good feature to support ephemeral feeds too, for coordinating replication swarms or distributed computing jobs.

**Comment by noffle** [comment]
Yes! This ties into a use case me and @karissa discussed was being able to annotate feeds & expose that in the pre-sync metadata, so you could see which feeds are marked as e.g. "media feeds" (biiiig data) and those could be avoided.
